content/docs/dvclive/get-started.md

```diff
@@ -118,23 +118,42 @@ training progress:
 
 <toggle>
 
-<tab title="Standalone report">
+<tab title="Standalone">
 
-At the end of each `step`, DVCLive will generate or update a report in
+By default, DVCLive will generate or update a report in
 [`Live.report_file`](/doc/dvclive/api-reference/live#properties) displaying all
-the logged data.
+the logged data:
 
 ![HTML report](/img/dvclive-html.gif)
 
 </tab>
 
+<tab title="Notebook">
+
+If you pass `report="notebook"` to DVCLive, the HTML report will be displayed
+and updated inside the output of the cell:
+
+![Notebook report](/img/dvclive-notebook.gif)
+
+</tab>
+
 <tab title="DVC Extension for VS Code">
 
 The
 [DVC Extension for VS Code](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc)
-will display all the data logged by DVCLive.
+will display all the data logged by DVCLive:
+
+![VS Code Report](/img/dvclive-vscode-monitoring.gif)
+
+</tab>
+
+<tab title="Iterative Studio">
 
-![VS Code monitoring](/img/dvclive-vscode-monitoring.gif)
+If you follow the instructions in
+[Studio Live Experiments](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots),
+DVCLive updates will be displayed in the Studio UI:
+
+![Studio Report](/img/dvclive-studio.gif)
```

Comment:
The live metrics line in the plot in this gif starts at step 9. I think that the live update of this line would be clearer to the user if it started at 3rd or 4th step and continued for another 3-4 steps.

Also, my understanding is that we should upload all the images/gifs used in the docs to the [static repo](https://github.com/iterative/static). Don't we use that for the DVCLive docs assets?

Reply:
> The live metrics line in the plot in this gif starts at step 9. I think that the live update of this line would be clearer to the user if it started at 3rd or 4th step and continued for another 3-4 steps.

👍 will update

> Also, my understanding is that we should upload all the images/gifs used in the docs to the [static repo](https://github.com/iterative/static). Don't we use that for the DVCLive docs assets?

I was not aware of that. Previously (and this P.R) I added images to https://github.com/iterative/dvc.org/tree/main/static/img . Did something change? Where is the process described?

Reply:
> Did something change? Where is the process described?

In my very first docs PR, @casperdcl made this suggestion [here](https://github.com/iterative/dvc.org/pull/2455#pullrequestreview-655796850) and [here](https://github.com/iterative/dvc.org/pull/2455#pullrequestreview-657475731). I believe it was discussed and decided prior to that.

Reply:
@jorgeorpinel Can you weigh in about how to add images? I've also never heard of this before 😅 .

Reply:
not sure what the current status is either. AFAIK (at least in iterative.ai/blog) we "do the right thing" and use DVC for the images.

Reply:
We only add images via DVC in the blog on iterative.ai, all docs images are generally popped into the repo plainly under the `/static/img` directory. Almost certainly not best practice, but until we get the process fully sorted out in the blog we've opted not to change things for other images since the blog easily has the lion's share of images and much more capacity to grow.